app/models/venue.rb

```diff
@@ -19,8 +19,8 @@ class Venue < ActiveRecord::Base
   belongs_to :user
 
   include PgSearch
-  pg_search_scope :search_by_venue_name,
-                  against: :venue_name,
+  pg_search_scope :search_by_venue,
+                  against: [:venue_name, :street_name, :state, :zip_code, :city, :description],
```

Comment:
Line is too long. [95/80]<br>Align the parameters of a method call if they span more than one line.